src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -119,9 +119,9 @@ public static class CommandLineParser
                     {
                         switch (directive.Key)
                         {
-                            case "vscode":
-                            case "jupyter":
-                            case "synapse":
+                            case "debug":
+                                break;
+                            default:
```

Comment:
Oh. This needs to be a strict allow list. The number of possible directives that we don't want to report is effectively infinite.

Reply:
ok then previous Pr is good